rai_test_utils/README.md

```diff
@@ -0,0 +1,11 @@
+# Responsible AI test utilities for Python
+
+### This package has been tested with Python 3.6, 3.7, 3.8 and 3.9
+
+The Responsible AI Utilities package contains common testing utilities and functions shared across various RAI tools, including fairlearn, interpret-community, responsibleai, raiwidgets, ml-wrappers and other packages.
```

Comment:
nit:
The Responsible AI **Test** Utilities package...

Reply:
done